### PR TITLE
add export to PDF

### PR DESCRIPTION
**Pull request by kolyaratishin**
I have done issue #97

### CODE REVIEW

File: api/src/main/java/com/eleks/academy/pharmagator/controllers/ExportController.java [hunk 1]
```diff
@@ -29,4 +38,21 @@ public void export(HttpServletResponse response) {
 
         workbook.close();
     }
+
+    @SneakyThrows
+    @GetMapping("/pdf")
+    public ResponseEntity<ByteArrayResource> exportToPDF() {
+        byte[] bytes = pdfExportService.export().toByteArray();
```
Comment (sergvasylchak) [inline api/src/main/java/com/eleks/academy/pharmagator/controllers/ExportController.java:45]: Make the service return `byte[]` or `ByteArrayResource`

**Review by sergvasylchak — APPROVED** [review]

File: api/build.gradle [hunk 2]
```diff
@@ -48,7 +48,7 @@ dependencies {
     implementation 'org.modelmapper:modelmapper:2.4.4'
     implementation 'org.apache.poi:poi:5.1.0'
     implementation 'org.apache.poi:poi-ooxml:5.1.0'
-
+    implementation 'com.itextpdf:itextpdf:5.5.13.2'
```
Comment (sergvasylchak) [inline api/build.gradle:51]: As far as I'm aware itext is not free-to-use in production

Comment (kolyaratishin) [reply api/build.gradle:51]: Should I change library?

Comment (sergvasylchak) [reply api/build.gradle:51]: Rather investigate the tradeoffs

**Pull request closed without merging**